mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -604,6 +632,52 @@ def log_metric(self, run_id, metric):
             if just_created:
                 self._update_latest_metric_if_necessary(logged_metric, session)
 
+    def log_metrics(self, run_id, metrics, session=None, check_run=True):
```

Comment:
Nit: Can we make the new functions introduced by this PR "private" by prepending an underscore, since these methods aren't part of the `AbstractStore` API and don't need to be called by external consumers?

```suggestion
    def _log_metrics(self, run_id, metrics, session=None, check_run=True):
```

Reply:
Sure, thanks for the suggestion